eth-reference-tests/src/referenceTest/java/tech/pegasys/teku/reference/phase0/forkchoice/ForkChoiceTestExecutor.java

```diff
@@ -137,7 +136,7 @@ spec, new SignedBlockAndState(anchorBlock, anchorState)),
             new ForkChoiceStateProvider(eventThread, recentChainData),
             new TickProcessor(spec, recentChainData),
             transitionBlockValidator,
-            DEFAULT_FORK_CHOICE_LATE_BLOCK_REORG_ENABLED,
+            true,
```

Comment:
why now hardcoded `true`?

Reply:
reference tests have late block reorg in them, and i evaluate that as to whether to do late block reorg ForkChoice, so if this isnt set true here, we fail late block reorg group basically.

Reply:
nit: maybe a comment explaining why we have the hardcoded value?